### DISCUSSION TITLE
Are you supposed to be able to walk, turn around whilst using a tool?

### DISCUSSION THREAD

**Discussion by sneedsfeedandseed** [Raicuparta/nomai-vr#427] [Q&A]
Because I can't. The right controller is for walking, the left for turning around. So when I hold the scanner in that hand, I can't use the joystick for movement. So I have to awkwardly stand still whilst scanning.

Is this intended?

Thanks.

**Comment by Raicuparta** [comment]
Yep, intended. I know it's not ideal but that's how it works right now.

**Reply by Raicuparta** [reply]
This isn't an issue in the current versions, now that tools have gesture buttons
